src/hooks/useFetchWalletTokens.ts

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect } from "react";
+import {
+  useConnection,
+  useWallet,
+} from "@solana/wallet-adapter-react";
+import {
+  AccountInfo,
+  Connection,
+  ParsedAccountData,
+  PublicKey,
+} from "@solana/web3.js";
+import { TOKEN_PROGRAM_ID } from "@solana/spl-token";
+
+type AccountResponse = {
+  pubkey: PublicKey;
+  account: AccountInfo<ParsedAccountData | Buffer>;
+};
+
+const getAccounts = (
+  connection: Connection,
+  publicKey: PublicKey
+): Promise<AccountResponse[]> =>
+  connection.getParsedProgramAccounts(TOKEN_PROGRAM_ID, {
```

Comment:
Any reason you used this function versus `getTokenAccountsByOwner`?

Reply:
Good call - updated in https://github.com/raindrop-studios/creator-ui/pull/3/commits/eeb4609811fc01d9f1e35ffc184316e9814b9238